tc-health-client/tmagent/tmagent.go

```diff
@@ -624,7 +638,7 @@ func (c *ParentInfo) readStrategies(parentStatus map[string]ParentStatus) error
 		return nil
 	}
 
-	log.Debugf("loading %s\n", fn)
```

Comment:
soading

Reply:
@zrhoffman ?

Reply:
Sorry, the review didn't grab the added line too. The *l* in *loading* was changed to an *s*:

https://github.com/apache/trafficcontrol/blob/ce3659d9098f3eaba033a97f3a0045ec45dd55c1/tc-health-client/tmagent/tmagent.go#L641